### ISSUE TITLE
Missing crafting recipes

### ISSUE THREAD

**Issue by tunikakeks** [PowerNukkitX/PowerNukkitX#1370]
# 🐞 I found a bug

### 📸 Screenshots / Videos
Example recipes:
![image](https://github.com/PowerNukkitX/PowerNukkitX/assets/54219265/01e47a82-48dd-497e-a238-ac3c1e69e753)
![image](https://github.com/PowerNukkitX/PowerNukkitX/assets/54219265/0be2f985-879c-4a19-ba49-495853fb3096)
![image](https://github.com/PowerNukkitX/PowerNukkitX/assets/54219265/a58e8123-3354-4d53-b412-f41dd14eab88)

### ▶ Steps to Reproduce
Try to craft wooden logs into planks / Try to craft quartz pillars

### ✔ Expected Behavior
Quartz pillars would get crafted insteadof another quartzblocks &
Planks should be the same type as the log (Spruce -> Spruce)

### ❌ Actual Behavior
The quartz pillar recipe gives you 2 quartzblocks and planks aren't the same type, most of them are resulting in oak (Spruce -> Oak)

* PowerNukkit version: 1.20.10-r1

**Comment by Verox001** [comment]
Thanks for the report. Will fix it in the next pull request.

**Comment by AzaleeX** [comment]
I'm thinking of working on this `craft problem` **tomorrow**.

**Comment by Verox001** [comment]
@tunikakeks Unfortunately I cannot confirm that crafting quartz pillars does result in two quartz blocks. Please provide more information if this problem persists for you or try updating PNX.
The bug with wrong wood types is fixed in the PR above.

**Comment by tunikakeks** [comment]
@Verox001 I just tried your fix but it don't work on my end. I used your changes to build pnx and started my server with it
![image](https://github.com/PowerNukkitX/PowerNukkitX/assets/54219265/a40a4d06-851d-41f3-a57c-ad902aa3702d)

**Comment by Verox001** [comment]
You probably compiled a wrong version or something like that. Let someone accept my PR first. And then make sure there is nothing wrong with your server (maybe you're starting it from a wrong entry file).

**Comment by tunikakeks** [comment]
I just checked it, it's weird. On a new server it works fine, but when I try it on my old server with the updated PNX.jar, it doesnt work (its the same jar file and the same start process)

**Comment by Verox001** [comment]
Yes, this seems more like a problem only occuring to you. @smartcmd May you be so kind accepting the PR and then marking this issue as solved, since everything else are probably deprecated errors?

**Comment by AzaleeX** [comment]
Yes, I'm going to do that because this problem is solved.
